### ISSUE TITLE
ERROR - prefect.Flow | Unexpected error occured in FlowRunner: TypeError("_add_version() got an unexpected keyword argument 'many'")

### ISSUE THREAD

**Issue by JoMiguelSantos** [PrefectHQ/prefect#1212]
Deploying the flow below gives this error. Since this is a very simple flow, not sure where the problem is :/

Some context: I'm running this flow on a Docker container locally, Python 3.7.2

<img width="1224" alt="Screen Shot 2019-07-08 at 11 24 25" src="https://user-images.githubusercontent.com/35111886/60799207-0513b780-a173-11e9-9056-92d54b7e5caf.png">

```"""Trigger model train."""
import logging
from prefect import Flow
from prefect import task
from prefect.schedules import IntervalSchedule
import datetime
import time
import pendulum
import utils.setup as setup
from datetime import timedelta
from datalib.logger import build_logger_object
import subprocess


DEFAULT_LOGGER = logging.getLogger("datalib-logger")
DEFAULT_LOGGER.addHandler(logging.NullHandler())

# Specify state handler / notification
def log_state_transition(flow, old_state, new_state):
    """Print message when new_state is entered."""
    msg = f"Flow/task {flow.name} transitioned from {old_state.serialize()['type']} into state {new_state.serialize()['type']}"
    # log
    try:
        DEFAULT_LOGGER.info(
            {
                "message": "started",
                "msg_flow": msg,
                "flow_name": str(flow.name),
                "old_state": str(old_state.serialize()["type"]),
                "old_state_message": old_state.serialize()["message"],
                "new_state": str(new_state.serialize()["type"]),
                "new_state_message": new_state.serialize()["message"].replace(
                    "'", "''"
                ),
            }
        )
    except:
        print("logging was not successful")
    return new_state


# Specify all tasks to be performed


@task(state_handlers=[log_state_transition])
def run_daily_models():
    res = subprocess.run(
        ["dbt run --models tag:daily"],
        shell=True,
        check=True,
        timeout=5,
        capture_output=True,
    )


@task(state_handlers=[log_state_transition])
def test_daily_models():
    res = subprocess.run(
        ["dbt test --models tag:daily"],
        shell=True,
        check=True,
        timeout=5,
        capture_output=True,
    )


@task(state_handlers=[log_state_transition])
def run_realtime_models():
    res = subprocess.run(
        ["dbt run --models tag:realtime"],
        shell=True,
        check=True,
        timeout=5,
        capture_output=True,
    )


@task(state_handlers=[log_state_transition])
def test_realtime_models():
    res = subprocess.run(
        ["dbt test --models tag:realtime"],
        shell=True,
        check=True,
        timeout=5,
        capture_output=True,
    )


# Define run schedule
daily_schedule = IntervalSchedule(
    start_date=pendulum.parse("2019-07-05"),
    interval=datetime.timedelta(days=1),
    end_date=pendulum.parse("2100-01-01"),
)

realtime_schedule = IntervalSchedule(
    start_date=pendulum.parse("2019-07-05"),
    interval=datetime.timedelta(minutes=2),
    end_date=pendulum.parse("2100-01-01"),
)


with Flow(
    "daily flow of running and testing dbt views",
    schedule=daily_schedule,
    state_handlers=[log_state_transition],
) as daily_flow:
    run_daily_models = run_daily_models(flow=daily_flow)
    test_daily_models = test_daily_models(flow=daily_flow)

with Flow(
    "realtime flow of running and testing dbt views",
    schedule=realtime_schedule,
    state_handlers=[log_state_transition],
) as realtime_flow:
    run_realtime_models = run_realtime_models(flow=realtime_flow)
    test_realtime_models = test_realtime_models(flow=realtime_flow)


# Run the flows with the associated schedule

if __name__ == "__main__":
    if setup.db_logger:
        logger = build_logger_object(
            setup.service_name,
            setup.section,
            setup.elk_host,
            setup.elk_port,
            logs_db=setup.db_logger,
            db_log_table="logs",
        )
    else:
        logger = build_logger_object(
            setup.service_name, setup.section, setup.elk_host, setup.elk_port
        )
    realtime_flow.run()
    daily_flow.run()
```

**Comment by joshmeek** [comment]
Hi @JoMiguelSantos what version of Prefect are you running? I believe this is a known issue w/ the marshmallow dependency.

**Comment by JoMiguelSantos** [comment]
Hey, I have on my requirements.txt `prefect>=0.5.5` so it should be the last version, no? I checked and I do have marshmallow on my dependencies. Any known fix I can use?

**Comment by joshmeek** [comment]
Can you update your version by pip installing off of the master branch because this was fixed in #1151.

`pip install git+https://github.com/PrefectHQ/prefect`

**Comment by JoMiguelSantos** [comment]
It works. Thank you!!
